scala3/scaladoc.md

```diff
@@ -54,16 +54,84 @@ For more information you can follow this [thread](https://contributors.scala-lan
 ![](../resources/images/scala3/scaladoc/snippet-compiler2.gif)
 ![](../resources/images/scala3/scaladoc/snippet-compiler1.gif)
 
-### Type-signature browsing
+### Inkuire (Hoogle-like searches)
 
-Haskell programmers are probably familiar with Hoogle - a documentation search engine that allows you browse the docs by signatures rather than symbolic names. Since many Scala developers are also functional programming fans, we decided to add a similar functionality to Scaladoc. The whole tool is based on [Inkuire](https://github.com/VirtusLab/Inkuire), which works with Kotlin and is being currently adapted to Scala 3.  
+Haskell programmers are probably familiar with [Hoogle](https://hoogle.haskell.org/) - a documentation search engine that allows you to find functions by their signatures rather than their symbolic names. Since many Scala developers are also functional programming fans, we decided to add a similar functionality to Scaladoc.
 
-![](../resources/images/scala3/scaladoc/inkuire.gif)
+To use this feature simply type the signature of the function You are looking for in scaladoc searchbar. This is how it works:
 
-To be up-to-date with this feature you can follow this [thread](https://contributors.scala-lang.org/t/pre-sip-scaladoc-search-by-type-signature/1604/15).
-Note that this feature is in an early stage of development. We still don't support type lambdas, higher-kinded types, and some other Scala features.
+![](../resources/images/scala3/scaladoc/inkuire-1.0.0-M2_js_flatMap.gif)
 
+This feature is provided by [Inkuire](https://github.com/VirtusLab/Inkuire) search engine, which works for Scala 3 and Kotlin. To be up-to-date with development of this feature follow [Inkuire repository](https://github.com/VirtusLab/Inkuire) for new releases.
 
+Note that this feature is still in development, so it can be subject to considerable change.
+If You encounter a bug or have an idea for improvement don't hesitate to create an issue on [Inkuire](https://github.com/VirtusLab/Inkuire/issues/new) or [dotty](https://github.com/lampepfl/dotty/issues/new).
+
+
+Using this feature allows to find `def`s and `val`s from the documented code.
+
+In order for a scaladoc searchbar query to be searched using Inkuire instead of the default search engine, the query has to contain `=>` character sequence. 
```

Comment:
```suggestion
In order for a scaladoc searchbar query to be searched using Inkuire instead of the default search engine, the query has to contain the `=>` character sequence. 
```

Reply:
done